src/Pages/LandingPage/LandingPage.jsx

```diff
@@ -8,10 +8,23 @@ import Footer from '../../Shared/Footer/Footer';
 import CategoryCard from '../../Shared/CategoryCard/CategoryCard';
 import {categories,topCategories, featuredProducts, testimonials, steps} from '../../Data';
 import blitheLogo from '../../Assets/Images/Blithe-logo.png';
+import axios from 'axios';
+import useAxios from '../../Api/useAxios/useAxios';
+import { getCategories } from '../../Api/actions';
+import { Link } from 'react-router-dom';
 
 const LandingPage = () =>{
 
-    console.log("testimonials", testimonials);
+    let { response, loading, error } = useAxios(getCategories);
+    const [data, setData] = useState([]);
+
+    useEffect(() => {
+        if (response !== null) {
+            setData(response);
+        }
```

Comment:
Suggestion: You can use response && setData(response)

Reply:
ok!